Assets/Scripts/System/Classes/AttractItem.cs

```diff
@@ -0,0 +1,87 @@
+﻿using UnityEngine;
+using System;
+using UnityEngine.UI;
+using System.IO;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+
+[System.Serializable]
+public class AttractItem {
+
+    public enum AttractItemType { Video, Image, Text }
+    public AttractItemType type;
+
+    public Sprite sprite;
+    public string text;
+
+    public string pathToItem;
+    public float displayTime = 10;
+
+    public bool voidItem = false;
+
+    /// <summary>
+    /// Constructs an AttractItem to be used later.
+    /// </summary>
+    /// <param name="pathToItem">Path to file, must be absolute.</param>
+    /// <returns>If the construction was successful.</returns>
+    public AttractItem (string filePath)
+    {
+        pathToItem = filePath;
+
+        if (pathToItem.Contains("mp4"))
+        {
+            type = AttractItemType.Video;
+            //Video player only requires a path to the video, so we don't need to load anything new here
+        }
+        else if (pathToItem.Contains("png") || pathToItem.Contains("jpg") || pathToItem.Contains("jpeg"))
+        {
+            type = AttractItemType.Image;
+            LoadImage();
+        }
+        else if (pathToItem.Contains("txt"))
+        {
+            type = AttractItemType.Text;
+            LoadText();
+        }
+        else
+        {
+            Debug.Log("AttractItem - No valid file found.  Voiding file " + pathToItem);
```

Comment:
Using `GM.Instance.logger` gives us the extra features of sending to the remote service (if configured) and control over which log levels are outputted.

Reply:
Oo yeah I always forget to change them over.  I always just use the built in Unity one because when I'm debugging a specific script, I just want those messages so I find it easier to just do that and switch over after.